DASH-IF-IOP.bs.md

```diff
@@ -10,8 +10,500 @@ Live discussion in #document-authoring on Slack.
+
+Clients SHOULD NOT present a [=media segment=] twice when it occurs on both sides of a period transition in a period-connected [=representation=].
+
+Clients SHOULD ensure seamless playback of period-connected [=representations=] in consecutive [=periods=].
+
+Note: The exact mechanism that ensures seamless playback depends on client capabilities and will be implementation-specific. Any shared [=media segment=] overlapping the [=period=] boundary may need to be detected and deduplicated to avoid presenting it twice.
+
+### Period continuity ### {#timing-connectivity-continuity}
+
+In addition to [[#timing-connectivity|period connectivity]], [[!MPEGDASH]] defines period continuity, which is a special case of period connectivity where the two samples on the boundary between the connected [=representations=] are consecutive on the same [=sample timeline=].
+
+Note: The above can only be true if the sample boundary exactly matches the period boundary.
+
+Period continuity MAY be signaled in the [=MPD=] when the above condition is met, in which case period connectivity SHALL NOT be simultaneously signaled on the same [=representation=]. Continuity implies connectivity.
+
+The signaling of period continuity is the same as for [[#timing-connectivity|period connectivity]], except that the value to use for `@schemeIdUri` is `urn:mpeg:dash:period-continuity:2015`.
+
+Clients MAY take advantage of any platform-specific optimizations for seamless playback that knowledge of period continuity enables; beyond that, clients SHALL treat continuity the same as connectivity.
+
+## Time shift window ## {#timing-timeshift}
+
+TSBD
+SPD
+
+Issue: Determine appropriate content for this section.
+
+## Real time clock synchronization ## {#timing-sync}
+
+Issue: Determine appropriate content for this section.
+
+## XLink ## {#timing-xlink}
+
+Issue: Determine appropriate content for this section.
+
+## Leap seconds ## {#timing-leapseconds}
+
+This section is intentionally left blank to indicate that the leap seconds topic is out of scope of this proposal.
+
+## Forbidden techniques ## {#timing-nonos}
+
+Some aspects of [[!MPEGDASH]] are not compatible with the interoperable timing model defined in this document. In the interest of clarity, they are explicitly listed here:
+
+* The `@presentationDuration` attribute SHALL NOT be used.
+
+## Dynamic MPD updates ## {#timing-mpd-updates}
+
+Dynamic MPDs MAY change over time. The nature of the change is not restricted unless such a restriction is explicitly defined.
+
+Some common reasons to make changes in dynamic [=MPDs=]:
+
+* Adding new [=media segments=] to an existing [=period=].
+* Adding new [=periods=].
+* Adjusting the durations and start times of [=periods=].
```

Comment:
* I'm not sure adjusting start times of periods makes sense. Shouldn't the start time of a period always remain the same?
* I don't think duration adjustment makes sense either, except in the case of adding a duration to the last period, where previously there was none. This may change if https://github.com/Dash-Industry-Forum/DASH-IF-IOP/issues/224 is acted upon, although even in that case the scenarios where it should be allowed should probably be quite restrictive.

Perhaps for now it would make sense to have this bullet point only cover adding a period duration? The issue box below ref's #224.

Reply:
I agree that start time adjustment should not be allowed. In fact, in the list below there is a SHALL NOT about it :) Will fix this list.

Will make it also require extend-only behavior for Period@duration.